node/src/components/storage/tests.rs

```diff
@@ -388,293 +357,346 @@ fn switch_block_for_block_header(
         Default::default(),
         Default::default(),
         Default::default(),
-        finalized_block,
+        finalized_block.clone(),
         Some(validator_weights),
         block_header.protocol_version(),
-        MERKLE_TREE_HASH_ACTIVATION.into(),
+        merkle_tree_hash_activation,
     )
     .expect("Could not create block")
 }
```

Comment:
```suggestion
}

```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2531/commits/1b1ab632ad8a3b673597c109ba986ccfb5638d5e